src/qt3utils/nidaq/customcontrollers.py

```diff
@@ -0,0 +1,168 @@
+import abc
+import logging
+import nidaqmx
+import numpy as np
+import time
+
+logger = logging.getLogger(__name__)
+
+
+class WavelengthControlBase(abc.ABC):
```

Comment:
What is the purporse of this controller? please explain

Reply:
The idea is that any PLE scan will have a method of controlling the wavelength. In the tests I have been doing, that controller is a voltage controller. But in your experiment you will be having a different wavelength control. All methods of controlling wavelength should share some common characteristics: min, max, get current value, etc. Thus a base class, which future wavelength controller can inherit from. Would you have any recommendations for improving it?

Reply:
This is great, please put a description based on this in the class docstring